### DISCUSSION TITLE
When to indicate the non-standard port?

### DISCUSSION THREAD

**Discussion by DarkCorner-IT** [fail2ban/fail2ban#3726] [Q&A] [answered]
Fail2ban filters the logs to find out if there is a fraudulent access attempt, but the port must be indicated in the configuration.
Therefore, it should be indifferent to leaving port = ssh instead of, for example, putting port =10222 if this is the port used for ssh accesses.
Can you explain to me if and when it is necessary to change the port in jail.local?

**Comment by sebres** [comment, marked as answer]
One needs to add `port = ...` to jail in the cases:
- where the port of daemon got changed (e. g. in `/etc/ssh/sshd_config`), however the alias (e. g. `ssh` in `/etc/services`) remains unchanged as usually.
- where the daemon becomes accessible over some additional port
- but only if the banning action is multiport action (not allports action), as it regularly the case for sshd jail by default

In any case after the change the jail (or fail2ban) needs to be restarted (to affect the chain/rules in the net-filter subsystem).
